products.json

```diff
@@ -2,6 +2,17 @@
   {
     "vid": 1,
     "name": "LIFX",
+    "defaults": {
+      "color": false,
+      "chain": false,
+      "matrix": false,
+      "relays": true,
+      "buttons": true,
```

Comment:
Why is buttons and relays true here in the default?

Reply:
oh, good catch!